trade-finance-manager-ui/server/types/view-models/utilisation-reports-view-model.ts

```diff
@@ -0,0 +1,22 @@
+import { IsoMonthStamp } from '@ukef/dtfs2-common';
+import { BaseViewModel } from './base-view-model';
+import { UtilisationReportReconciliationSummaryItem } from '../utilisation-reports';
+
+export type SummaryItemViewModel = UtilisationReportReconciliationSummaryItem & {
```

Comment:
Seeing as we are exporting these types, could we make the names a bit more specific than just `Summary<Thing>`?